CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Unreleased
 ### New features
 
 ### Bug fixes
+- Remove stray semicolon in CSS
```

Comment:
Please add a link to this PR.  If you wouldn't mind fixing the one for EOI (#506) at the same time, I'd appreciate it.